constructr/src/main/scala/de/heikoseeberger/constructr/ConstructrMachine.scala

```diff
@@ -286,6 +280,12 @@ final class ConstructrMachine(
       goto(retryState)
   }
 
+  // Unhandled events
+
+  whenUnhandled {
+    case _: Event => stay()
```

Comment:
I don't want to be too nit-picking, but out of curiosity: are there any messages other than `Event`s delivered to a `FSM`? If not a simple wildcard pattern is all we need.